web-animations-2/Overview.bs

```diff
@@ -2353,6 +2376,152 @@ Items sorted earlier are executed before those sorted later.
 
 </ins>
 
+Animation Triggers {#triggers}
+-----------------
+
+### Overview ### {#triggers-overview}
+
+An <dfn export>animation trigger</dfn> is used to control the playback
+of its associated [=animation=] for time-driven animations.
+Like animations, [=animation triggers=] are associated with a [=timeline=]
+and are attached to an [=animation attachment range|attachment range=].
+
+Issue: Should there be any effect of triggers on scroll-driven animations?
+
+### Animation Trigger Active Interval ### {#trigger-active-interval}
+
+[=Animation triggers=] define an <dfn for="animation trigger">active interval</dfn>,
+which is the period of time during which the trigger is scheduled to produce its effect
+on its associated [=animation=]’s playback.
+Each [=animation trigger=] has only one such interval,
```

Comment:
This is super confusing, having *one* interval and then referring to two ranges. I think we should say that the active interval is determined by its most recent state.

Reply:
OK, I've rephrased the text to be more elaborate and explicit regarding dependency on state

Reply:
Actually, I'm refactoring all of it, and I'll add that as well.